pkg/controller/controller.go

```diff
@@ -551,6 +576,13 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 		pv.Spec.PersistentVolumeSource.CSI.FSType = fsType
 	}
 
+	if performInTreeTranslation {
```

Comment:
`performInTreeTranslation` doesn't seem like the right name.

It relates to "this specific volume is using translation logic for migration"
`volumeMigrated`?
`volumeShimmed`?
`volumeTranslated`?

I would go with translated probably but will defer to you